src/NodeAutoCompleteViewExtension/NodeAutoCompleteViewExtension.cs

```diff
@@ -119,28 +119,25 @@ public override void Loaded(ViewLoadedParams viewLoadedParams)
             this.viewLoadedParamsReference = viewLoadedParams ?? throw new ArgumentNullException(nameof(viewLoadedParams));
             var dynamoViewModel = viewLoadedParams.DynamoWindow.DataContext as DynamoViewModel;
 
-            if (dynamoViewModel.IsDNAClusterPlacementEnabled)
+            DependencyView = new NodeAutoCompleteView(this, viewLoadedParams);
```

Comment:
So the side panel only makes sense for the cluster so I guess it should continue to be dependent on IsDNAClusterPlacementEnabled ?
If so then I guess we should hide the button in the bar that makes the side panel visible as well.